src/pyenphase/updaters/production.py

```diff
@@ -18,7 +23,9 @@ class EnvoyProductionUpdater(EnvoyUpdater):
     allow_inverters_fallback = False
     working_endpoints: list[
         str
-    ] = []  # list of successful production endpoint to use by inverters fallback
+    ] = (
```

Comment:
This still needs to be created in `__init__` to be safe so the list isn't shared

Reply:
No need anymore as class variable. Made it local variable in probe method.